org.eclipse.jdt.core/codeassist/org/eclipse/jdt/internal/codeassist/complete/CompletionOnReferenceExpressionName.java

```diff
@@ -44,10 +44,14 @@ public TypeBinding resolveType(BlockScope scope) {
 			this.resolvedTypeArguments = new TypeBinding[length];
 			for (int i = 0; i < length; i++) {
 				TypeReference typeReference = this.typeArguments[i];
-				if ((this.resolvedTypeArguments[i] = typeReference.resolveType(scope, true /* check bounds*/)) == null) {
+				if ((this.resolvedTypeArguments[i] = typeReference.resolveType(scope,
+						true /* check bounds */)) == null) {
 					typeArgumentsHaveErrors = true;
 				}
-				if (typeArgumentsHaveErrors && typeReference instanceof Wildcard) { // resolveType on wildcard always return null above, resolveTypeArgument is the real workhorse.
+				if (typeArgumentsHaveErrors && typeReference instanceof Wildcard) { // resolveType on wildcard always
+																					// return null above,
+																					// resolveTypeArgument is the real
+																					// workhorse.
```

Comment:
who's auto-formatting unchanged code?

Reply:
I will fix it, I'm manually formatting the code since the auto formatter sometimes change unwanted lines which are int he edge of a modification. And i time to time use vscode with jdt.ls for development with the eclipse default formatter.
I will be vigilant in future to make sure with white space issues since I have been make bit a lot last few PRs :(.